### PR TITLE
Lecture 07-a

### PR DESCRIPTION
**Pull request by dykim1222**
Optimization is one of the important aspects of deep learning. We talk about updating parameters by gradient descent and tuning the learning rate. Choosing a right step size may reduce training time. With proper learning rate, the gradient doesn’t paddle left or right, and this also leads to the convergence to the global minimizer in quadratic case.

-- First part of Lecture 7 (25 March 2019) by Yash Goel and Marvin Singson Mananghaya. Edited by Dae Young Kim.

### CODE REVIEW

**Comment by Atcold** [comment]
Thanks!

**Comment by Atcold** [comment]
Hmm, I think there's a typo.
`$\frac{1}{p} \sum_p$` should be `$\frac{1}{P} \sum_{p=1}^P$`, I think.

**Comment by Atcold** [comment]
Also, you just covered half the optimisation chapter. I mean, you wrote two sections. Two more are missing. And they were quite interesting.

**Comment by dykim1222** [comment]
Hi Alfredo,

-Typo: I can fix it. Do I fix it on my fork and then make another pull
request?
-Last message: Quite confused. Can you elaborate more on what are missing?

Thanks
Dae

**Comment by Atcold** [comment]
Pull the latest `master` in your repo (or simply clone it again, if you prefer), and fix the typo. Then you can send a new PR.

Anything before 1:07 in the video should be covered in these notes.
Missing section 0:39 (cost function appearance and eigendecomposition).
Missing section 0:57 (example with several input characteristics combination).
Missing section 1:07 (maybe this should be covered by the other group) (how to fix these issues).

**Comment by dykim1222** [comment]
Got it. We will write/edit it and send a new PR soon. Thank you.

**Comment by Atcold** [comment]
That would be _really_ nice. I think you may want to do it _before_ tomorrow exam...

**Comment by dykim1222** [comment]
Hi Alfredo,

We will work on the first two topics if that is okay. We started working on
them but not sure about before exam...
